puppet/zulip/manifests/memcached.pp

```diff
@@ -35,8 +44,8 @@
   }
   file { '/etc/sasl2/memcached-sasldb2':
     require => Exec[generate_memcached_sasldb2],
-    owner   => 'memcache',
-    group   => 'memcache',
+    owner   => $memcached_group,
```

Comment:
This should be `$memcached_user` or `$memcached_user_group` or something.

Reply:
Renamed into `$memcached_user`.